packages/runtime/src/api/react.ts

```diff
@@ -57,15 +44,15 @@ export class MakeswiftHostApiClient {
   graphqlClient: GraphQLClient
   makeswiftApiClient: MakeswiftApiClient.Store
   subscribe: MakeswiftApiClient.Store['subscribe']
-  private localizedResourcesMap: LocalizedResourcesMap
-  private locale: Intl.Locale | undefined
 
-  constructor({ uri, cacheData, localizedResourcesMap = {}, locale }: MakeswiftClientOptions) {
+  constructor({ uri, cacheData, locale }: { uri: string; cacheData?: CacheData; locale?: string }) {
```

Comment:
Note that `locale` is a string now.

Reply:
I didn't see a good reason to construct a `Intl.Locale` instance that we always convert to back a language string anyway. If we want to type the `locale` argument, we can brand the string type.

Reply:
That's fair. We were trying to use "parsed" types like `Intl.Locale` or `URL` whenever we were using a string that could be better represented with one of these platform APIs. I think that's stronger than a branded type, in particular because a branded type is only useful once you've done some sort of validation that what you've got is validated or parsed and it can be branded. `new Intl.Locale()` or `parseLocale(locale: string): BrandedLocale` somewhat achieve the same thing but it feels better to "use the platform".

I think right now the validation for locales is "does calling `Intl.Locale` throw?" and we want to validate user-provided values somehow. We might change it to something else (we've had some issues with `Intl.Locale`, like it changing `en-UK` to `en-GB`) but when we do it will have to be across the board (i.e., the builder and API as well).

Reply:
I guess I don't buy into using "parsed" objects as a better representation of primitive values in a React app where the default method of value comparison is `Object.is`. I'd rather have a validated primitive value up until the point where we need the "parsed" type functionality, if there is such a point.